pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -282,13 +282,14 @@ self: super: ({
     '' + drv.postPatch or "";
   }) super.foldl;
 
+  # https://github.com/fpco/inline-c/issues/127
+  inline-c-cpp = appendPatch ./patches/inline-c-cpp-fcompact-unwind.patch super.inline-c-cpp;
+
 } // lib.optionalAttrs pkgs.stdenv.isAarch64 {  # aarch64-darwin
 
   # https://github.com/fpco/unliftio/issues/87
   unliftio = dontCheck super.unliftio;
 
-  # https://github.com/fpco/inline-c/issues/127
-  inline-c-cpp = dontCheck super.inline-c-cpp;
```

Comment:
Can you add this override back in `configuration-ghc-8.10.x.nix` and `configuration-8.8.x.nix`? The package would keep working there then.